.golangci.yml

```diff
@@ -1,16 +1,10 @@
 run:
   timeout: 15m
-  skip-files:
-    - ^.*\.(pb|y)\.go$
-  skip-dirs:
-    - "vendor$"
-    - "pkg/app/piped/executor/analysis/mannwhitney"
 
 linters:
   disable-all: true
   enable:
     - depguard
-    - exportloopref
```

Comment:
golangci-lint tells me this rule is deprecated and not necessary.

`copyloopvar` is unnecessary because this rule tells us the copy of the loop variable, which we can delete.

```
WARN The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar.
```